build_minecraft_bot.sh

```diff
@@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
 else
     cd catkin_ws/src
     # Add the package
-    ln -s ../../minecraft_bot/ minecraft_bot
+    catkin_create_pkg minecraft_bot std_msgs rospy roscpp
+    cp -R ../../minecraft_bot .
+    # ln -s ../../minecraft_bot/ minecraft_bot    Symlinks do not work in virtualbox natively
```

Comment:
OK I figured out a fix.  Using the vagrant file in opencog/opencog

You need to run virtualbox as administrator
 and then run this at cmd:
VBoxManage setextradata opencog-dev-vm
VBoxInternal2/SharedFoldersEnableSymlinksCreate//vagrant 1


Send a new commit to put back the symlink.


On Mon, Jan 11, 2016 at 12:07 AM, Amen Belayneh <notifications@github.com>
wrote:

> In build_minecraft_bot.sh
> <https://github.com/opencog/opencog-to-minecraft/pull/17#discussion_r49290517>
> :
>
> > @@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
> >  else
> >      cd catkin_ws/src
> >      # Add the package
> > -    ln -s ../../minecraft_bot/ minecraft_bot
> > +    catkin_create_pkg minecraft_bot std_msgs rospy roscpp
> > +    cp -R ../../minecraft_bot .
> > +    # ln -s ../../minecraft_bot/ minecraft_bot    Symlinks do not work in virtualbox natively
>
> If the Vagrantfile is particular to this project would you pls include it
> in the pr?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog-to-minecraft/pull/17/files#r49290517>.
>